libflux/flux/src/cffi.rs

```diff
@@ -1,4 +1,10 @@
-use std::{ffi::*, mem, os::raw::c_char};
+use std::{
+    any::Any,
+    ffi::*,
+    mem,
+    os::raw::c_char,
+    panic::{catch_unwind, resume_unwind},
```

Comment:
When this import style gets nested, I would prefer we use another line for it. It makes it hard to read as it gets more and more complicated, whereas single lines make it clear where something comes from. In this case, can we put `use std::panic::{catch_unwind, resume_unwind};` on a separate line?

Reply:
Out of laziness (I guess) I tend to prefer keeping imports nested like this since it is easier to add and remove imports. I did a batch reformat a while back but I suppose we could do the inverse of that if that is generally prefered https://github.com/influxdata/flux/pull/4213/files